.docker/web/run.sh

```diff
@@ -7,10 +7,8 @@ if [ -z $TOOLFORGE_DOCKER_PORT ]; then
 fi
 
 # Install dependencies.
-apt-get update
-apt-get install librsvg2-bin
 composer install -d /var/www || exit 1;
```

Comment:
Since you've set the working directory, I don't think `-d /var/www` is necessary.

Reply:
Good point; removed. I'm not sure why we were bothering to exit either; if there's an issue it'll become apparent, surely.